server/models/Profile.js

```diff
@@ -1,39 +1,42 @@
-const mongoose = require("mongoose");
+const mongoose = require('mongoose');
 
 const profileSchema = new mongoose.Schema({
   userId: {
     type: mongoose.Schema.Types.ObjectId,
     required: true,
-    ref: 'User'
+    ref: 'User',
   },
   name: {
     type: String,
-    default: "",
+    default: '',
   },
   description: {
     type: String,
-    default: "",
+    default: '',
   },
   gender: {
     type: String,
     enum: ['male', 'female', 'other'],
   },
   address: {
     type: String,
-    default: "",
+    default: '',
   },
   telephone: {
     type: String,
-    default: "",
+    default: '',
   },
   birthday: {
     type: Date,
-    default: null
+    default: null,
   },
   photo: {
     type: String,
-    default: "",
+    default: '',
+  },
+  cloudinary_id: {
```

Comment:
Super close on this one - could we just make it camelCase. Everything else looks great! Make sure to change the other files to use `cloudinaryId`